Makefile

```diff
@@ -76,7 +76,7 @@ u_boot_armv8: $(O) $(D) optee tfa
 	$(Q)$(MAKE) -C $(UBOOT_DIR) CROSS_COMPILE=$(CROSS_COMPILE_64) O=$(O)/u-boot/armv8 $(UBOOT_ARMV8_DEFCONFIG)
 	$(Q)$(MAKE) -C $(UBOOT_DIR) CROSS_COMPILE=$(CROSS_COMPILE_64) O=$(O)/u-boot/armv8 BINMAN_INDIRS=$(FW_DIR) \
 					BL31=$(I)/bl31.bin \
-				  TEE=$(I)/tee-pager_v2.bin
+					TEE=$(I)/tee-pager_v2.bin
```

Comment:
Probably a good time to clean this up:
 Since OP-TEE 3.21.0 this could/should be replaced with tee-raw.bin. Although tee-pager_v2.bin is still produced unconditionally (even when the "pager" feature is disabled in OP-TEE) it might not be the case in the future, and the name is somewhat misleading when pager is disabled anyways (which is clearly the case here since tee-pageable_v2.bin is not used).

Reply:
Works for me, will add rename as part of this fixup.